src/renderers/shared/fiber/ReactFiberStack.js

```diff
@@ -34,6 +34,21 @@ exports.createCursor = function<T>(defaultValue : T) : StackCursor<T> {
   };
 };
 
+exports.getPrevious = function<T>(
+  cursor : StackCursor<T>,
+) : T | null {
+  if (index < 0) {
+    if (__DEV__) {
+      warning(false, 'Unexpected read of previous.');
+    }
+    return null;
+  }
+  if (index === 0) {
+    return null;
+  }
+  return valueStack[index - 1];
```

Comment:
PS I was already thinking about doing this, but I believe I'll submit a follow-up PR that adds some unit tests for `ReactFiberStack` itself.